src/manifest/clog.sidecar

```diff
@@ -22956,7 +22984,7 @@
+    "CustomTypeClogCSharpFileContents": "/*++\r\n\r\n    Copyright (c) Microsoft Corporation.\r\n    Licensed under the MIT License.\r\n\r\n--*/\r\n\r\nusing System;\r\nusing System.Text;\r\nusing System.Net;\r\nusing System.Runtime.InteropServices;\r\n\r\nnamespace msquic.clog_config\r\n{\r\n    public class Types\r\n    {\r\n        public static string ADDR(byte[] value)\r\n        {\r\n            int si_family = value[0] | ((ushort)value[1] << 8);\r\n            int sin_port = value[2] | ((ushort)value[3] << 8);\r\n\r\n            Span<byte> sa2 = value;\r\n\r\n            string msg = \"\";\r\n\r\n            switch (si_family)\r\n            {\r\n                case 0:  //<--unspecified\r\n                    msg += $\"*:{sin_port}\";\r\n                    break;\r\n                case 2:  //< --v4\r\n                    msg += $\"{new IPAddress(sa2.Slice(4, 4)).ToString()}:{sin_port}\";\r\n                    break;\r\n                case 10:  //<--v6 (linux)\r\n                case 23: //<--v6\r\n                    msg += $\"[{new IPAddress(sa2.Slice(8, 16)).ToString()}]:{sin_port}\";\r\n                    break;\r\n                default:\r\n                    throw new NotSupportedException(\"Invalid SI_FAMILY : \" + si_family);\r\n            }\r\n            \r\n            return msg;\r\n        }\r\n\r\n        public static string CID(byte [] value)\r\n        {\r\n            StringBuilder hex = new StringBuilder(value.Length * 2);\r\n            foreach (byte v in value)\r\n            {\r\n                hex.AppendFormat(v.ToString(\"X2\"));\r\n            }\r\n            return hex.ToString();\r\n        }\r\n    }\r\n}\r\n",
```

Comment:
Ditto for updating clog and reverting this change.

Reply:
I merged master locally (accept incoming), and when I re-ran clog this change went away. so I don't think I need it.